### PR TITLE
Address warnings from CI build (update github actions) 

### PR DESCRIPTION
**Pull request by lavanyaj3**
Address warnings from CI build (update github actions) 
Fixes: #106

### CODE REVIEW

**Comment by fbricon** [comment]
Please don't remove the comments

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -10,56 +9,58 @@ on:
...
+        if: runner.os == 'Linux'
+        run: |
+          # Install Xvfb
+          sudo apt-get update
+          sudo apt-get install -y xvfb
+          # Start Xvfb
+          Xvfb :99 -screen 0 1920x1080x24 &
+          export DISPLAY=:99
+          # Run your tests
+          npm test
```
Comment (fbricon) [inline .github/workflows/build.yaml:56]: whole setup is unnecessary, as xvfb is available on ubuntu runners now. 

See: 

- https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md#installed-apt-packages
- https://github.com/coactions/setup-xvfb/issues/21#issuecomment-2021625936

File: .github/workflows/build.yaml [hunk 2]
```diff
@@ -10,56 +9,58 @@ on:
...
+
+      - name: Run Test on macOS
+        if: runner.os == 'macOS'
```
Comment (fbricon) [inline .github/workflows/build.yaml:59]: runner.os != 'Linux'

we'll need to run tests on windows (but that's another issue)

Comment (lavanyaj3) [reply .github/workflows/build.yaml]: ## Run tests
      - name: Run Tests
        run: |
          if [ "${{ runner.os }}" == "Linux" ]; then
            xvfb-run npm test
          else
            npm test
          fi
 
 Made the changes accordingly

**Review by fbricon — APPROVED** [review]

**Comment by fbricon** [comment]
thanks @lavanyaj3

**Pull request merged**